### PR TITLE
sriov: added policy unit tests

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/sriov/policy.go [hunk 1]
```diff
@@ -108,6 +112,10 @@ func (builder *PolicyBuilder) WithVFRange(firstVF, lastVF int) *PolicyBuilder {
 		return builder
 	}
 
+	if firstVF < 0 || lastVF < 0 {
+		builder.errorMsg = "firstPF and lastVF can not be less than 0"
```
Comment (cdvultur) [inline pkg/sriov/policy.go:116]: Based on condition this should be:
```suggestion
		builder.errorMsg = "firstPF or lastVF can not be less than 0"
```

**Review by cdvultur — COMMENTED** [review]
Minor comment

Otherwise lgtm

File: pkg/sriov/policy.go (hunk 1 above)
Comment (achuzhoy) [reply pkg/sriov/policy.go:116]: firstVF (not PF) according to the condition

File: pkg/sriov/policy_test.go [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+		apiClient,
+		defaultPolicyName,
+		defaultPolicyNsName,
+		defaultPolicyResName,
+		defaultPolicyVFNum,
+		defaultPolicyNICs,
+		defaultPolicyNodeSelector)
+}
+
+// // buildValidTestBuilder returns a valid Builder for testing purposes.
```
Comment (sebrandon1) [inline pkg/sriov/policy_test.go:480]: Adjust comment here for `buildInvalidSriovPolicyTestBuilder`.

**Review by sebrandon1 — CHANGES_REQUESTED** [review]
Small function comment

File: pkg/sriov/policy_test.go [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+		},
+		{
+			firstVF:           0,
+			lastVF:            65,
+			expectedErrorText: "lastVF can not be greater than 63",
+		},
+		{
+			firstVF:           65,
+			lastVF:            1,
+			expectedErrorText: "firstPF argument can not be greater than lastPF",
```
Comment (achuzhoy) [inline pkg/sriov/policy_test.go:264]: Why first PF if you used firstVF ?

File: pkg/sriov/policy_test.go [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+		},
+		{
+			firstVF:           -2,
+			lastVF:            0,
+			expectedErrorText: "firstPF or lastVF can not be less than 0",
```
Comment (achuzhoy) [inline pkg/sriov/policy_test.go:269]: same

File: pkg/sriov/policy_test.go [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+	testCases := []struct {
+		mtu               int
+		expectedErrorText string
+	}{
+		{
+			mtu:               1500,
+			expectedErrorText: "",
+		},
+		{
+			mtu:               0,
```
Comment (achuzhoy) [inline pkg/sriov/policy_test.go:295]: imho this or the next test is redundant.

**Pull request merged**